recommendation/microsoft.owin.hosting.json

```diff
@@ -0,0 +1,126 @@
+﻿{
+  "Name": "Microsoft.Owin.Hosting",
+  "Version": "1.0.0",
+  "Packages": [
+    {
+      "Name": "Owin",
+      "Type": "Nuget"
+    }
+  ],
+  "Recommendations": [
+    {
+      "Type": "Namespace",
+      "Name": "Microsoft.Owin.Hosting",
+      "Value": "Microsoft.Owin.Hosting",
+      "KeyType": "Name",
+      "ContainingType": "",
+      "RecommendedActions": [
+        {
+          "Source": "Amazon",
+          "Preferred": "Yes",
+          "TargetFrameworks": [
+            {
+              "Name": "netcoreapp3.1",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            },
+            {
+              "Name": "net5.0",
+              "TargetCPU": [ "x86", "x64", "ARM32", "ARM64" ]
+            }
+          ],
+          "Description": "Add reference to Microsoft.AspNetCore.Hosting\r\nAdd reference to Microsoft.AspNetCore.Server.Kestrel\r\nRemove Microsoft.Owin.Hosting\r\nRemove Owin\r\n",
```

Comment:
Description is what displays in the PA tool. We need to update this to be a sentence with no newlines. Please check other descriptions for examples.
This applies to all high level descriptions

Reply:
Thanks, all done.